lib/ipam/ipam_test.go

```diff
@@ -186,6 +186,9 @@ var _ = testutils.E2eDatastoreDescribe("IPAM tests", testutils.DatastoreAll, fun
 		})
 
 		Measure("It should be able to allocate a single address quickly - blocksize 32", func(b Benchmarker) {
+			origLevel := log.GetLevel()
```

Comment:
Rather than do this within each Measure() block, you can set the log level in the `BeforeEach` call above, and then revert it to the original value in `AfterEach()`

Reply:
Right, I missed that, will do! Thanks, @caseydavenport !